homeassistant/components/shelly/manifest.json

```diff
@@ -9,7 +9,7 @@
   "iot_class": "local_push",
   "loggers": ["aioshelly"],
   "quality_scale": "platinum",
-  "requirements": ["aioshelly==8.1.1"],
+  "requirements": ["aioshelly==8.2.0"],
```

Comment:
If its not a breaking change we usually do the library bump in its own PR in case we want to backport it later or there is a fix after it that requires backporting it as well